src/bioregistry/data/bioregistry.json

```diff
@@ -29993,6 +30304,12 @@
     "uri_format": "http://purl.obolibrary.org/obo/ECOCORE_$1"
   },
   "ecocyc": {
+    "contact": {
+      "email": "pkarp@ai.sri.com",
+      "github": "pkarp111",
+      "name": "Peter Karp",
+      "orcid": "0000-0002-5876-6418"
+    },
```

Comment:
Is the contact entry supposed to be under the ecocyc key? Shouldn't it be under the main entry?